cli/src/cmd/exec.rs

```diff
@@ -350,6 +371,55 @@ impl Cli {
                 print!("{}", diff);
                 Ok(ExitCode::SUCCESS)
             }
+            Subcommand::CallPlugin(identifier, function, input_arg, output) => {
+                let mut input = input_arg.as_bytes().to_vec();
+
+                // if the value is a path to a file
+                if input_arg.to_owned().starts_with("@") {
+                    let path = input_arg
+                        .chars()
+                        .skip(1)
+                        .take(input_arg.len() - 1)
+                        .collect::<String>();
+                    input = tokio::fs::read(Path::new(&path)).await?;
+                }
+
+                let client = Client::new(self.host.as_str())?;
+                let res = client.call_plugin(identifier, function, input).await?;
+
+                if let Some(output) = output {
+                    tokio::fs::write(output, res).await?;
+                } else {
+                    std::io::stdout().write_all(&res);
+                }
+
+                Ok(ExitCode::SUCCESS)
+            }
+            Subcommand::InstallPlugin(identifier, name, location, wasm) => {
+                // TODO: implement location handling
+                if let Some(wasm) = wasm {
+                    let wasm = tokio::fs::read(wasm).await?;
+                    let name = match name {
+                        Some(v) => v,
+                        None => "",
+                    };
+                    let location = match location {
+                        Some(v) => v,
+                        None => "",
+                    };
```

Comment:
:eyes: Look at the Option type's unwrap_or_default or unwrap_or_else methods for these

Reply:
:raised_hands: thank you!